### PR TITLE
Add Etrian Odyssey IV: Legends of the Titan

### PR DESCRIPTION
**Pull request by findanother1**
Adds Etrian Odyssey IV: Legends of the Titan.

*Automatically generated by PR builder*

### CODE REVIEW

File: etrian-odyssey-iv-legends-of-the-titan/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Etrian Odyssey IV: Legends of the Titan"
+description = "A dungeon-crawler JRPG developed by ATLUS, where you travel across the overworld on a flying ship to explore dungeons and mark unique areas onto your map. You can create your very own guild of explorers with a wide selection of classes that come with unique sets of skills and abilities that would help along the journey and discover the mystery of Yggdrasil.\nFPS takes slight hit when loading into a zone / battle, and when GUI opens up, e.g. dialogue while in a dungeon."
```
Comment (TheKoopaKingdom) [inline etrian-odyssey-iv-legends-of-the-titan/game.dat]: The `\n` shouldn't be here.

Comment (TheKoopaKingdom) [inline etrian-odyssey-iv-legends-of-the-titan/game.dat]: >takes slight hit when loading into a zone / battle, and when GUI opens up, e.g. dialogue while in a dungeon.

`description` should only give info about the game, and not how it performs. This type of thing belongs in the wiki instead.

Comment (Schplee) [inline etrian-odyssey-iv-legends-of-the-titan/game.dat]: To add to what @TheKoopaKingdom said, the description should be sourced from Wikipedia, as it states in contributions guide.

**Comment by Schplee** [comment]
LGTM - will fix nits myself.

**Pull request merged**